package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "fluxo_root",
```

Comment:
Can we have workspace configured so that the development and deployment  will be easy. [Refer this](https://yarnpkg.com/blog/2017/08/02/introducing-workspaces/)

Reply:
Done